src/components/TransactionModal/TransactionModal.style.ts

```diff
@@ -123,7 +123,8 @@ export const Success = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
-  background: ${rgba(colors.blue[500], 0.2)};
+  /* TODO replace with variable: blue 500 */
+  background: ${rgba('#4038ff', 0.2)};
```

Comment:
I'm not really sure what the actionable item here is. Could you do a simple structure change here so that the background is a separate element and we can apply `opacity` to it?